hashicorp/dev-portal
# refactor: use MdxP element

## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Creates a new `MdxP` component, which replaces `.p` styles that had to be independently applied to our `Text` component for docs and tutorial content.

## 🤷 Why

To consolidate MDX content styling across docs and tutorial content into a single place: `components/dev-dot-content/mdx-components`.

## 🛠️ How

- Moves `dev-dot-content` `.p` styles into a new `MdxP` component in `components/dev-dot-content/mdx-components`
- Uses the `MdxP` component for docs and tutorials

## 🧪 Testing

- Visit a docs MDX page, such as [/vault/docs/get-started/developer-qs][/vault/docs/get-started/developer-qs], and confirm headings look as expected (vs [upstream][UPSTREAM/vault/docs/get-started/developer-qs])
- Visit a tutorial MDX page, such as [/waypoint/tutorials/get-started-docker/get-started-ui][/waypoint/tutorials/get-started-docker/get-started-ui], and confirm headings look as expected (vs [upstream][UPSTREAM/waypoint/tutorials/get-started-docker/get-started-ui])

## 💭 Anything else?

Not at the moment!

[task]: https://app.asana.com/0/1202097197789424/1202072894754378/f
[preview]: https://dev-portal-git-zsmdx-p-hashicorp.vercel.app/
[/vault/docs/get-started/developer-qs]: https://dev-portal-git-zsmdx-p-hashicorp.vercel.app/vault/docs/get-started/developer-qs
[UPSTREAM/vault/docs/get-started/developer-qs]: https://developer.hashicorp.com/vault/docs/get-started/developer-qs
[/waypoint/tutorials/get-started-docker/get-started-ui]: https://dev-portal-git-zsmdx-p-hashicorp.vercel.app/waypoint/tutorials/get-started-docker/get-started-ui
[UPSTREAM/waypoint/tutorials/get-started-docker/get-started-ui]: https://developer.hashicorp.com/waypoint/tutorials/get-started-docker/get-started-ui
